edm4hep.yaml

```
@@ -270,290 +270,290 @@ datatypes:
-      - float time                     //time of the hit [ns].
-      - float eDep                     //energy deposited on the hit [GeV].
-      - float eDepError                //error measured on EDep [GeV].
-      - edm4hep::Vector3d position     //hit position in [mm].
-      - std::array<float,6> covMatrix  //covariance of the position (x,y,z), stored as lower triangle matrix. i.e. cov(x,x) , cov(y,x) , cov(y,y) , cov(z,x) , cov(z,y) , cov(z,z)
+      - uint64_t cellID      // ID of the sensor that created this hit.
+      - int32_t type                       // type of raw data hit, either one of edm4hep::RawTimeSeries, edm4hep::SIMTRACKERHIT.
+      - int32_t quality                    // quality bit flag of the hit.
+      - float time                     // time of the hit [ns].
+      - float eDep                     // energy deposited on the hit [GeV].
+      - float eDepError                // error measured on EDep [GeV].
+      - edm4hep::Vector3d position     // hit position in [mm].
+      - std::array<float,6> covMatrix  // covariance of the position (x,y,z), stored as lower triangle matrix. i.e. cov(x,x) , cov(y,x) , cov(y,y) , cov(z,x) , cov(z,y) , cov(z,z)
     VectorMembers:
-      - edm4hep::ObjectID  rawHits     //raw data hits. Check getType to get actual data type.
+      - edm4hep::ObjectID  rawHits     // raw data hits. Check getType to get actual data type.
 
 
   #-------------  TrackerHitPlane
   edm4hep::TrackerHitPlane:
     Description: "Tracker hit plane"
-    Author: "Placido Fernandez Declara, CERN"
+    Author: "EDM4hep authors"
     Members:
-      - uint64_t cellID      //ID of the sensor that created this hit
-      - int32_t type                       //type of raw data hit, either one of edm4hep::RawTimeSeries, edm4hep::SIMTRACKERHIT - see collection parameters "TrackerHitTypeNames" and "TrackerHitTypeValues".
-      - int32_t quality                    //quality bit flag of the hit.
-      - float time                     //time of the hit [ns].
-      - float eDep                     //energy deposited on the hit [GeV].
-      - float eDepError                //error measured on EDep [GeV].
-      - edm4hep::Vector2f u            //measurement direction vector, u lies in the x-y plane
-      - edm4hep::Vector2f v            //measurement direction vector, v is along z
-      - float du                       //measurement error along the direction
-      - float dv                       //measurement error along the direction
-      - edm4hep::Vector3d position     //hit position in [mm].
-      - std::array<float,6> covMatrix  //covariance of the position (x,y,z), stored as lower triangle matrix. i.e. cov(x,x) , cov(y,x) , cov(y,y) , cov(z,x) , cov(z,y) , cov(z,z)
+      - uint64_t cellID      // ID of the sensor that created this hit.
+      - int32_t type                       // type of raw data hit, either one of edm4hep::RawTimeSeries, edm4hep::SIMTRACKERHIT.
```

```suggestion
      - int32_t type                       // type of raw data hit.
```
I don't think we define anything like that at the moment.